core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -483,6 +484,13 @@ class SpendReportingService(
         workspaceServiceConstructor(childContext).getGCPWorkspacesByBillingProjects(validWorkspaceIds)
       }
 
+  /* As long as no aggregation contains "daily", it is cacheable */
```

Comment:
I think we also need to exclude requests where the aggregation key is just `Category`, because the cache is aggregated by `Workspace~Category`

Reply:
Fixed. I made it so _only_ `Workspace~Category` is cacheable. Since that's what the UI uses for the landing page of each billing project, we can ensure that is cached.